fsck/cmd/getInfo.go

```diff
@@ -15,22 +15,72 @@
 package cmd
 
 import (
+	"bufio"
+	"bytes"
+	"encoding/binary"
 	"encoding/json"
 	"fmt"
 	"github.com/cubefs/cubefs/proto"
+	"github.com/cubefs/cubefs/util/errors"
```

Comment:
the import may need format
```go
// base package
// line break
// third-party package
// line break
// this project
```

Reply:
@pengtianyue025 Thanks for your review, I have fix it, please review again.